src/components/NearMe/NearMeProfiles.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react'
+import { Box, Grid, ImageList, ImageListItem, Typography } from '@mui/material'
+import { LocationOn } from '@mui/icons-material'
+import { makeStyles } from 'tss-react/mui'
+import { getUsersNearMeData } from '../../actions/userServices'
+import { picProfileColMob, picProfileColWeb } from './enums'
+
```

Comment:
Sorry, I missed the fact of namings. Types, enums - with capital letter. Its convention.

Reply:
Renamed the "enums" file to "Enums".